content/configuration/3.files.md

```diff
@@ -44,6 +44,10 @@ Based on your configured drivers, you must also provide additional variables, wh
 | `STORAGE_<LOCATION>_ACL`                    | S3 ACL.                    |                    |
 | `STORAGE_<LOCATION>_SERVER_SIDE_ENCRYPTION` | S3 server side encryption. |                    |
 | `STORAGE_<LOCATION>_FORCE_PATH_STYLE`       | S3 force path style.       | false              |
+| `STORAGE_<LOCATION>_CONNECTION_TIMEOUT`     | S3 connection timeout.     | 5000               |
+| `STORAGE_<LOCATION>_SOCKET_TIMEOUT`         | S3 socket timeout.         | 120000             |
```

Comment:
For the timeouts shall we add `in milliseconds` for clarity?

Reply:
Excellent point, thank you!